qmlbind/src/wrapper.h

```diff
@@ -12,18 +12,26 @@ class MetaObject;
 class Wrapper : public QObject
 {
 public:
-    Wrapper(const std::shared_ptr<const MetaObject> &metaObject, const Backref &ref);
+    Wrapper(const std::shared_ptr<const MetaObject> &metaObject, qmlbind_client_object *clientObject, qmlbind_interface_handlers handlers);
```

Comment:
`qmlbind_interface_handlers` is tied to `MetaObject`, so what about getting it from metaObject in the constructor and remove it from parameters?